src/common/KokkosKernels_Handle.hpp

```diff
@@ -48,9 +48,20 @@
 #include "KokkosSparse_spadd_handle.hpp"
 #include "KokkosSparse_sptrsv_handle.hpp"
 #include "KokkosSparse_spiluk_handle.hpp"
+
 #ifndef _KOKKOSKERNELHANDLE_HPP
 #define _KOKKOSKERNELHANDLE_HPP
 
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CBLAS)   && \
+    defined(KOKKOSKERNELS_ENABLE_TPL_LAPACKE) && \
+   (defined(KOKKOSKERNELS_ENABLE_TPL_SUPERLU) || \
+    defined(KOKKOSKERNELS_ENABLE_TPL_CHOLMOD))
+
+ // Enable supernodal sptrsv
+ #define KOKKOSKERNELS_ENABLE_SUPERNODAL_SPTRSV
+
+#endif
+
 namespace KokkosKernels{
 
 namespace Experimental{
```

Comment:
Given that Clark is planning on using this stuff, it sounds like the intent is for this to be production capability.  In that case, why put it in the Experimental namespace?  If it's enabled by default, you'll need to support it anyway -- at least you'll need to make the build work and tests pass.